spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -399,6 +404,11 @@ public Template getTemplate() {
 		 */
 		private String trustStorePassword;
 
+		/**
+		 * Trust stroe algorithm.
+		 */
+		private String trustStoreAlgorithm;
```

Comment:
This field's initial value should be `"SunX509"`, aligning it with Spring AMQP's default.